config/owasp/suppressions.xml

```diff
@@ -89,5 +89,14 @@
       <packageUrl regex="true">^(?!pkg:maven/org\.json/json@).+$</packageUrl>
       <cpe>cpe:/a:json-java_project:json-java</cpe>
     </suppress>
-  
+
+  <suppress>
+    <notes><![CDATA[
+   There is no fix, Google deprecated the method but did not remove it or fix it.
```

Comment:
this is in OSSIndex which we disable on Jenkins, but if you run the scan locally you'll see it so may as well suppress it